.github/workflows/deploy.yml

```
@@ -1628,19 +1634,19 @@ jobs:
                /usr/local/lib/libassimp*
                /usr/local/lib/libzlibstatic.a
-        key: ${{ runner.os }}-build-${{ env.cache-name }}
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ env.MACOS_CACHE_VERSION }}
         restore-keys: |
-          ${{ runner.os }}-build-${{ env.cache-name }}-
-      
+          ${{ runner.os }}-build-${{ env.cache-name }}-${{ env.MACOS_CACHE_VERSION }}-
+
     - name: Cache Ogre
       id: cache-ogre-macos
       uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)